helix-view/src/document.rs

```diff
@@ -162,6 +162,20 @@ impl Document {
         Ok(doc)
     }
 
+    pub fn reload(&mut self) -> Result<(), anyhow::Error> {
+        use std::{fs::File, io::BufReader};
+
+        let path = self
+            .path()
+            .ok_or(Error::msg("unable to reload document without path"))?;
+        let file = File::open(path).context(format!("unable to open {:?}", path))?;
+        let text = Rope::from_reader(BufReader::new(file))?;
```

Comment:
We may want to reuse the existing open logic since we handle missing `\n` there.

Reply:
Fixed in https://github.com/helix-editor/helix/pull/177/commits/e20d2c6e79745c39d3d9b086e43cdef6d05009a3.